app/controllers/mentees_controller.rb

```diff
@@ -8,4 +9,13 @@ def index
       render json: { mentees: mentees }, status: :ok
     end
   end
+
+  def add_course
+    current_mentee = Mentee.find(params[:id])
+    payload = JSON.parse(request.body.read)
+    current_mentee.update!(course_id: payload['course_id'])
```

Comment:
Could you just access `course_id` from `params` without parsing the request payload?